types/ali-app/tslint.json

```diff
@@ -0,0 +1,8 @@
+{
+	"extends": "dtslint/dt.json",
+	"rules": {
+		"no-mergeable-namespace": false,
+		"no-any-union": false,
```

Comment:
Any reason we can't just disable this rule?

Reply:
Like some string union type:

```ts
interface I {
  type: 'type1' | 'type2' | string;
}
```

I prefer use some type add `any`

```ts
interface I {
  data: string | any;
}
```

anyway, I removed this rule.

Reply:
Gotcha - I know what you mean and feel both ways about it sometimes even if it doesn't totally make sense. :)